judo/app/visualization.py

```diff
@@ -4,13 +4,16 @@
 import warnings
 
 import mujoco
+import numpy as np
 import pyarrow as pa
 import viser
 from dora_utils.dataclasses import from_arrow, to_arrow
 from dora_utils.node import DoraNode, on_event
 from omegaconf import DictConfig
+from PIL import Image
 from viser import GuiFolderHandle, GuiInputHandle, MeshHandle
 
+from judo import REPOSITORY_ROOT
```

Comment:
as mentioned on slack, we need to be extremely careful referencing paths outside the package structure within the package, because those assets are not shipped with the wheel when uploading to pypi. we should either ensure that extra assets are shipped with the package as part of the project, or we can just move the logo into the package (a lot dirtier)